### ISSUE TITLE
Doesnt work on 6.2.4

### ISSUE THREAD

**Issue by wennaspeedy** [Digitalone1/PulseEffects-Presets#4]
Preset doesnt work with pipewire and easyeffect 6.2.4. Opening preset doesnt do anything - no plugin shows.

**Comment by Digitalone1** [comment]
I can't reproduce on my system. Everything is working. Did you follow the instructions correctly? 

Which distribution are you using? Are you on Flatpak?

**Comment by MilkyMad** [comment]
Hello there!
Doesn't show/load plugins on EasyEffects 6.2.5, Fedora36-rpm. Header 0.3.49, Library 0.3.52. LoudnessEqualizer.json for EasyEffects on Pipewire, moved to output folder. It is present in Presets, but nothing happening after pressing Load.

**Comment by Digitalone1** [comment]
> Hello there! Doesn't show/load plugins on EasyEffects 6.2.5, Fedora36-rpm. Header 0.3.49, Library 0.3.52. LoudnessEqualizer.json for EasyEffects on Pipewire, moved to output folder. It is present in Presets, but nothing happening after pressing Load.

Please post here the content of the preset file you saved. I suspect it may have been saved in a wrong format.

**Comment by MilkyMad** [comment]
> Please post here the content of the preset file you saved. I suspect it may have been saved in a wrong format.

Yep, you're right. Just copied code in file in raw, all good. Thanks. Content of saved file was with html parts.

My mistake, downloaded the file through RMB...
